### PR TITLE
[otbn] Extend testplan as discussed in testplan review meeting

### PR DESCRIPTION
**Pull request by rswarbrick**

### CODE REVIEW

File: hw/ip/otbn/data/otbn_testplan.hjson [hunk 1]
```diff
@@ -69,5 +77,14 @@
...
       tests: ["otbn_multi"]
     }
+    {
+      name: stress_all
+      desc: '''
+            Run multiple sequences back-to-back while making invalid TL
+            accesses at the same time.
+            '''
+      milestone: V2
+      tests: ["uart_stress_all"]
```
Comment (weicaiyang) [inline hw/ip/otbn/data/otbn_testplan.hjson:87]:
```suggestion
      tests: ["otbn_stress_all"]
```

**Review by weicaiyang — APPROVED** [review]
can you also add how you verify the idle? otherwise, LGTM

**Review by sriyerg — APPROVED** [review]

**Review by cindychip — APPROVED** [review]

File: hw/ip/otbn/data/otbn_testplan.hjson (hunk 1 above)
Comment (rswarbrick) [reply hw/ip/otbn/data/otbn_testplan.hjson:87]: Oops! (Guess where I copy-pasted from... :-))

**Comment by rswarbrick** [comment]
@weicaiyang:
> can you also add how you verify the idle?

Good point. I'll add something to the DV document and push it up in a separate PR.

**Pull request merged**